digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/ProjectLinkDaoSpec.scala

```diff
@@ -45,21 +45,11 @@ class ProjectLinkDaoSpec  extends FunSuite with Matchers {
     sqlu"""insert into project (id,state,name,ely,created_by, start_date) VALUES (2,0,'testproject2',1,'automatedtest', sysdate)""".execute
   }
 
-  test("Add two links that are not reserved") {
-    OracleDatabase.withDynTransaction {
-      addprojects()
-      /*Insert links to project*/
-      sqlu"""insert into project_link (id,project_id,track_code,discontinuity_type,road_number,road_part_number,start_addr_M,end_addr_M,lrm_position_id,created_by) VALUES (1,1,1,0,1,1,1,1,20000286,'automatedtest')""".execute
-      sqlu"""insert into project_link (id,project_id,track_code,discontinuity_type,road_number,road_part_number,start_addr_M,end_addr_M,lrm_position_id,created_by) VALUES (2,2,1,0,1,1,1,1,20000287,'automatedtest')""".execute
-      sql"""SELECT COUNT(*) FROM project_link WHERE created_by = 'automatedtest'""".as[Long].first should be(2L)
-      dynamicSession.rollback()
-    }
-  }
-
   //TRIGGER has been removed for the time being. Should activate the test again if the triggers are activated again.
```

Comment:
should this comment be removed if test is no longer ignored?

Reply:
Yes